articles/quickstart/webapp/aspnet-core/interactive.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Add Login to your ASP.NET MVC application
+description: This tutorial demonstrates how to add user login to an ASP.NET Core application.
+budicon: 448
+topics:
+  - quickstarts
+  - webapp
+  - aspnet-core
+  - login
+github:
+  path: Quickstart/Sample
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/startup
+  - files/appsettings
+  - files/login
+  - files/user-profile
+  - files/logout
+  - files/account.controller
```

Comment:
This PR doesn't include all these files, guessing just need to remove the ones that aren't included.